workitem/link/link_repository.go

```diff
@@ -279,3 +283,36 @@ func (r *GormWorkItemLinkRepository) Save(ctx context.Context, lt app.WorkItemLi
 	result := ConvertLinkFromModel(res)
 	return &result, nil
 }
+
+// ListWorkItemChildren get all child work items
+func (r *GormWorkItemLinkRepository) ListWorkItemChildren(ctx context.Context, parent string) ([]*app.WorkItem, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitem", "children", "query"}, time.Now())
+	wilt := WorkItemLinkType{}
+	r.db.Where("forward_name = ?", "relates to").First(&wilt)
```

Comment:
For now, that is hard-coded as per the story ( https://github.com/fabric8io/fabric8-planner/issues/884 ).